website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Adding fonts
+description: An intro to adding fonts to your project.
+slug: adding-fonts
+authors: iobrado
+date: 2022-04-12
+tags: [eightshift, boilerplate, fonts]
+hide_table_of_contents: false
+---
+
+Each project is different in its way. Logo, various fonts, etc. are what define the visual identity of your website. In this post, we'll cover adding fonts to a project.
+<!--truncate-->
+
+### Importing fonts into your project
```

Comment:
I see that this is the case in previous blog posts as well, but I believe that this should be an `h2`, not an `h3`